app/controllers/admin/bulk_republishing_controller.rb

```diff
@@ -37,6 +74,6 @@ def enforce_permissions!
   end
 
   def build_republishing_event(action:, bulk_content_type:)
-    RepublishingEvent.new(user: current_user, reason: params.fetch(:reason), action:, bulk_content_type:, bulk: true)
+    RepublishingEvent.new(user: current_user, reason: params.fetch(:reason), action:, bulk_content_type:, bulk: true, content_type: params.fetch(:content_type, nil))
```

Comment:
And here, maybe it's better to pass in the upper camelcase content type we get in the action rather than using the kebabified version from the params